hr_employee_holiday/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'HR Employee Holidays',
+    'version': '12.0.1.0.0',
+    'category': 'Human Resources',
+    'website': 'https://github.com/OCA/hr',
+    'author':
+        'Brainbean Apps, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'application': False,
+    'summary': 'Allows reviewing employee holidays via employee form',
+    'depends': [
+        'hr',
```

Comment:
@alexey-pelykh no need to here, because of `hr_holidays` depends on `hr`

Reply:
I got your point, yet I'm not 100% sure about transient dependencies - sometimes other reviews mentioned that it's needed to include those. @pedrobaeza bring in some light please - since I had doubt even when wrote that line

Reply:
I think, in case no direct data from hr module is used (specific view, security group. etc) you can remove the dependency.